src/utils/hotspotNftsUtils.ts

```diff
@@ -0,0 +1,93 @@
+import * as client from '@helium/distributor-oracle'
+import { LazyDistributor } from '@helium/idls/lib/types/lazy_distributor'
+import { IDL } from '@helium/idls/lib/esm/lazy_distributor'
+import {
+  init,
+  lazyDistributorKey,
+  PROGRAM_ID,
+} from '@helium/lazy-distributor-sdk'
+import { toNumber, MOBILE_MINT } from '@helium/spl-utils'
+import { Program, setProvider } from '@project-serum/anchor'
+import { getMint } from '@solana/spl-token'
+import { PublicKey } from '@solana/web3.js'
+import BN from 'bn.js'
+import { useState } from 'react'
+import { JsonMetadata, Metadata, Metaplex } from '@metaplex-foundation/js'
+import { useAsync } from 'react-async-hook'
+import { Recipient } from './useRecipient'
+import { useAnchorProvider } from './useAnchorProvider'
+
+export const LAZY_KEY = lazyDistributorKey(new PublicKey(MOBILE_MINT))[0]
+
+export function useProgram() {
+  const [program, setProgram] = useState<Program<LazyDistributor> | null>(null)
+  const anchorProvider = useAnchorProvider()
+  useAsync(async () => {
+    if (!anchorProvider) return
+    setProvider(anchorProvider)
+
+    const p = await init(anchorProvider, PROGRAM_ID, IDL)
+    setProgram((prog) => prog || (p as unknown as Program<LazyDistributor>))
+  }, [anchorProvider])
+
+  return program
+}
+
+export async function getPendingRewards(
+  program: Program<LazyDistributor>,
+  mint: PublicKey,
+  maybeRecipient: Recipient | undefined,
+) {
+  const lazyDistributor = await program.account.lazyDistributorV0.fetch(
+    LAZY_KEY,
+  )
+
+  const oracleRewards = await client.getCurrentRewards(
+    // TODO: Fix program type
```

Comment:
can we fix it now?

Reply:
Would like to fix this once I add the compressed NFTs